data-raw/clean_data.R

```diff
@@ -133,6 +134,79 @@ lifestage_lookup <- tibble(lifestage = unique(catch_2021_raw$lifestage),
+deer_catch_2022_raw <- read_csv("data-raw/deer_2022_catch.csv") |>
+  mutate(date = as.Date(date))
+catch_2022_raw <- bind_rows(mill_catch_2022_raw, deer_catch_2022_raw)
+
+# trap 2022
+mill_trap_2022_raw <- read_csv("data-raw/mill_2022_trap.csv") |>
+  mutate(date = as.Date(date),
+         time = as.character(time),
+         stream = ifelse(stream == "mill", "mill creek", stream))
+deer_trap_2022_raw <- read_csv("data-raw/deer_2022_trap.csv") |>
+  mutate(time = hms::as_hms(time),
+         time = as.character(time),
+         date = as.Date(date))
+trap_2022_raw <- bind_rows(mill_trap_2022_raw, deer_trap_2022_raw)
+
+# recaptures 2022
+mill_recaptures_2022_raw <- read_csv("data-raw/mill_2022_recaptures.csv")
+
+
+# 2022 lookup tables ------------------------------------------------------
+species_lookup_2022 <- tibble(species = unique(catch_2022_raw$species),
+                         species_clean = c("chinook salmon",
+                                           "rainbow trout",
+                                           "hard head",
+                                           "pacific lamprey",
+                                           "pacific lamprey",
+                                           "sacramento pikeminnow",
+                                           "prickly sculpin",
+                                           "riffle sculpin",
+                                           "river lamprey",
+                                           "ammocoete",
+                                           "sacramento sucker",
+                                           "speckled dace",
+                                           "prickly sculpin",
+                                           "california roach",
+                                           "cyprinid fry",
+                                           "riffle sculpin",
+                                           "california roach",
+                                           "california roach",
+                                           "pacific lamprey",
+                                           "hard head",
+                                           NA,
+                                           "pacific lamprey",
+                                           "rainbow trout",
+                                           "river lamprey",
+                                           "sacramento pikeminnow",
+                                           "california roach",
+                                           "sacramento sucker",
+                                           "rainbow trout",
+                                           "riffle sculpin",
+                                           "california roach",
+                                           "prickly sculpin",
+                                           "unknown sculpin",
+                                           "unknown fry"))
+lifestage_lookup_2022 <- tibble(lifestage = unique(catch_2022_raw$lifestage),
```

Comment:
Need to add the codes. My best guess is 1 = sac fry, 2= fry, 3 = parr, 4 = silvery parr, 5 = smolt. We should confirm with Ryan.